src/agent/samplers/cpu/linux/bandwidth/mod.bpf.c

```diff
@@ -200,8 +202,8 @@ int throttle_cfs_rq(struct pt_regs *ctx)
 
     // record throttle start time
     u64 now = bpf_ktime_get_ns();
-    u32 cgroup_idx = (u32)cgroup_id;
-    bpf_map_update_elem(&throttle_start, &cgroup_idx, &now, BPF_ANY);
+    u32 cgroup_runqueue_idx = (u32)cgroup_id * MAX_CPUS + cpu;
```

Comment:
My instinct is that CPU should be the outer key so that cachelines don't bounce around? 

Eg:
`cpu * MAX_CGROUPS + (u32)cgroup_id`